docs/build-the-docs.md

```diff
@@ -35,6 +35,33 @@ antora site.yml
 
 If all goes well, you will _not_ see any console output.
 
+#### Testing For Broken Xref Links
+
+There are a host of ways to check for broken links, including NPM's broken link checker.
+However, this custom generator, provided by the core Antora team, is able to check [the native Antora xref links](https://docs.antora.org/antora/1.0/asciidoc/page-to-page-xref/#xref-and-page-id-anatomy).
+It doesn't check external links, but it's still a good start.
+To use it, you need to pass the custom generator (in `./generator/xref-validator`) to the `generate` command, as in the example below.
+```console
+antora generate \
+    --pull \
+    --ui-bundle-url https://github.com/owncloud/docs-ui/releases/download/1.1.0/ui-bundle.zip \
```

Comment:
I would reference to the local ``ui-bundle.zip`` not the one from docs/releases

Reply:
I believe the release artifact is the right one, for the time being, until the build pipeline is finalised.

Reply:
sorry disagree. we currently do have regular updates in docs-ui and as long the automated process is not in place, any reference to an old bundle will counteract the updates made. just my 2c